config/config.go

```diff
@@ -318,9 +327,35 @@ func validate() error {
 		}
 	}
 
+	if vip.GetString(OperatorExtraDomainKey) != "" ||
+		vip.GetString(OperatorExtraIPKey) != "" {
+		if vip.GetString(ConnectUrlHost) != "" {
+			return fmt.Errorf("%s and %s/%s cannot be set together",
```

Comment:
imho they CAN be set together, we should check the hostname to be either OperatorExtraDomainKey or OperatorExtraIPKey.

Beaware that host != hostname, we should extract and left out the port. (AFAIK std library helps on this)

Reply:
Done